model/ManyManyList.php

```diff
@@ -230,37 +230,29 @@ public function add($item, $extraFields = array()) {
 				$hasExisting = false;
 			}
 
-			$manipulation = array();
+			// Blank manipulation
+			$manipulation = array(
+				$this->joinTable => array(
+					'command' => $hasExisting ? 'update' : 'insert',
+					'fields' => array()
+				)
+			);
 			if($hasExisting) {
-				$manipulation[$this->joinTable]['command'] = 'update';
 				$manipulation[$this->joinTable]['where'] = array(
 					"\"{$this->joinTable}\".\"{$this->foreignKey}\"" => $foreignID,
 					"\"{$this->joinTable}\".\"{$this->localKey}\"" => $itemID
 				);
-			} else {
-				$manipulation[$this->joinTable]['command'] = 'insert';
 			}
 
-			if($extraFields) {
```

Comment:
The existing code was trying to hack in custom manipulations, and it was breaking on sqlite3, so I simply took the code from DataObject and ensured it used the same method.